provider-service/base/Makefile

```diff
@@ -0,0 +1,24 @@
+##@ Development
+unit-test:
+	go test ./... -tags=unit
+
+decouple-test:
+	go test ./... -tags=decoupled
+
+test: unit-test decouple-test integration-test
```

Comment:
```suggestion
decoupled-test:
	go test ./... -tags=decoupled

test: unit-test decoupled-test integration-test
```

Reply:
correct changed in 847762891ac9e4c53e697be2fbd16d03bfb70260